src/bound.jl

```diff
@@ -10,8 +10,31 @@ const IntervalBound = Tuple{Float64,Float64}
 """
 $(TYPEDEF)
 
-Shortcut for possible bounds: either no bound is present (`nothing`), or a
-single number is interpreted as an exact equality bound, or a tuple of 2
-numbers is interpreted as an interval bound.
+A special type of bound where the variable may only take on the values 0 or 1
+exclusively.
 """
-const Bound = Union{Nothing,Float64,IntervalBound}
+struct BinaryBound end
+
+const Binary = BinaryBound()
```

Comment:
not a fan of the name (JuMP naming sucks hard here)

Reply:
(also docs)

Reply:
Yep, the naming was tough, especially `Integers`, which is too close to Julia's native `Integer` type I think

Reply:
Other options are `OneOrZero` and `EveryInterger` or something (JuMP has similar things as as MOI sets)